src/linear_algebra/matrix/determinant.lean

```diff
@@ -726,6 +728,10 @@ begin
   ring
 end
 
+@[simp] lemma det_fin_two_mk (a b c d : R) :
+  matrix.det ![![a, b], ![c, d]] = a * d - b * c :=
+det_fin_two _
```

Comment:
This will conflict in a minor way with #14992

Reply:
Indeed, this will need fixing in whichever PR is merged second.

Reply:
Do you think `matrix.of` in #14992 should be called `matrix.mk` instead? Either I should do a rename there, or this should be called
```suggestion
@[simp] lemma det_fin_two_of (a b c d : R) :
  matrix.det ![![a, b], ![c, d]] = a * d - b * c :=
det_fin_two _
```

Reply:
I think `matrix.mk` is a little bit more canonical, yes, but `of` looks also fine to me.